ego/bundle/main.go

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) Edgeless Systems GmbH.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//go:build linux
+// +build linux
+
+package main
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"debug/elf"
+	"io"
+	"os"
+	"os/exec"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+
+	"ego/cli"
+
+	"github.com/spf13/afero"
+)
+
+func main() {
+	// Open the current execeutable
```

Comment:
```suggestion
// Don't touch! Automatically injected at build-time.
var (
	version   = "0.0.0"
	gitCommit = "0000000000000000000000000000000000000000"
)

func main() {
	fmt.Fprintf(os.Stderr, "EGo (bundled) v%v (%v)\n", version, gitCommit)

	// Open the current execeutable
```
This was often helpful for me if someone asked for help and posted the output. You already inject the values in the CMake build step.

Reply:
Will add this, but I think we can refactor this to use Go 1.18's VCS integration at some point. Or are there any other advantages as to keep this CMake based approach? Though this would not be part of this PR.

Reply:
Probably no advantages for the CMake approach. However, currently we don't require Go 1.18 to build EGo. I suggest we postpone this until either there's a need for a change or we require Go 1.18 because of other reasons.

Reply:
True, I also experimented and we need to inject the version anyways or hard core it in some way, given that VCS does not seem to embed git tags. So yeah, not really worth it for now.